gradle/documentation/render-javadoc.gradle

```diff
@@ -210,9 +210,6 @@ class RenderJavadocTask extends DefaultTask {
   @Input
   def offlineLinks = [:]
 
-  @Input
-  def luceneDocUrl = "${-> project.luceneDocUrl }"
-
   @Input
   def solrDocUrl = "${-> project.solrDocUrl }"
```

Comment:
I think we should do here the same like in Lucene to allow snapshots without any link (null).

Reply:
Yup. I can't do all of it though. I've spent a good two hours cleaning up other cruft. ;)

Reply:
Updated the PR, seems to work.

Reply:
Looks great, 😊